### ISSUE TITLE
[Accessibility]: Simple bar chart bars with value 0 do not have aria-label

### ISSUE THREAD

**Issue by danielannan17** [carbon-design-system/carbon-charts#1839]
### Application/Team

API Connect

### Environment

Browser

### What happened? What did you expect to happen instead?

Simple bar chart bars with a value of 0 do not have an aria-label. They should either not be rendered or have an aria-label="0" similar to other bars which have their aria-labels set to their value.
![Violation](https://github.com/carbon-design-system/carbon-charts/assets/11006124/13ede1f9-bd90-47ab-921d-7f15c9be8c0f)

### What WCAG 2.1 checkpoint does the issue violate?

1.1.1

### Version

@carbon/charts-react@1.16.3

### What priority level would this issue be in your opinion?

P0
